### DISCUSSION TITLE
Any idea when ws dependency will be updated to @8.17.1?

### DISCUSSION THREAD

**Discussion by jberek-tricentis** [socketio/socket.io#5147] [Q&A]
Just wondering.

**Comment by darrachequesne** [comment]
Hi!

`socket.io@4.7.5` depends on `engine.io@~6.5.2` ([ref](https://github.com/socketio/socket.io/blob/4.7.5/package.json#L53)), so it should install `engine.io@6.5.5` due to the `~` range, which depends on `ws@8.17.1` ([ref](https://github.com/socketio/engine.io/blob/6.5.5/package.json#L44)).

Isn't that the case for you?
